app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new(params.require(:comment).permit(:title, :text))
+    @post = Post.find(params[:post_id])
+    @user = current_user
+    @comment.author = current_user
+    @comment.post_id = @post.id
+    respond_to do |format|
+      format.html do
+        if @comment.save
+          flash[:success] = 'Comment saved successfully'
+          redirect_to user_post_url(@user, @post)
+        else
+          flash.now[:error] = 'Error: Comment could not be saved'
+        end
+      end
+    end
+  end
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As I mentioned in the previous comment, I think it would be better if you use the **"[the strong parameters](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)"** like you studied in **[this lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)** for the project requirements